SmartDeviceLink/SDLMutableDataQueue.h

```diff
@@ -0,0 +1,38 @@
+//
+//  SDLMutableDataQueue.h
+//  
+//
+//  Created by David Switzer on 3/2/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@interface SDLMutableDataQueue : NSObject
```

Comment:
This class is very specifically used in conjunction with the IAP data session. We could decide to move it out of utilities and make it a helper class for the IAP transport.